src/photos/ducks/albums/index.jsx

```diff
@@ -25,13 +24,31 @@ const ALBUMS_QUERY = client =>
 export const ALBUM_QUERY = (client, ownProps) =>
   client.get(DOCTYPE_ALBUMS, ownProps.router.params.albumId).include(['photos'])
 
+const addPhotos = async (album, photos) => {
+  try {
+    const photoCountBefore = album.photos.data.length
+    await album.photos.addById(photos.map(({ _id }) => _id))
```

Comment:
Oh. Is it really `addById` and not `addByIds` ? :(

Reply:
It's `addById` for backwards compatibility reasons. But `addById (_id)` also works of course.